src/Core/Trending.php

```diff
@@ -10,23 +10,28 @@ class Trending
       'bitbucket'
     ];
 
-    public static function fetch($args) {
+    public static function fetch(array $args) 
```

Comment:
Since you touched this class, I want to mention that I have many questions about its implementation... But let's talk about them in a separate issue, believe me or not, I really want you to finalize this one! 😆

Reply:
I'm sure this won't be the last iteration though :laughing:  and i won't mind actually i love this